File: php/Aksesoris.php
<?php

//library
require('PetShop.php');

//deklarasi class Aksesoris yang menjadi anak class PetShop
class Aksesoris extends PetShop
{
     //private atribut
     private $jenis;
     private $bahan;
     private $warna;
     
     //constructor
     public function __construct($jenis = '', $bahan = '', $warna = '')
     {
         $this->jenis = $jenis;
         $this->bahan = $bahan;
         $this->warna = $warna;
     }
 
     //setter
     public function setJenis($jenis)
     {
         $this->jenis = $jenis;
     }
     public function setBahan($bahan)
     {
         $this->bahan = $bahan;
     }
     public function setwarna($warna)
     {
         $this->warna = $warna;
     }
     
     //getter
     public function getJenis()
     {
         return $this->jenis;
     }
     public function getbahan()
     {
         return $this->bahan;
     }
     public function getwarna()
     {
         return $this->warna;
     }
     
}
?>
?>

File: java/Aksesoris.java
class Aksesoris extends PetShop
{
    //atribut private
    private String jenis;
    private String bahan;
    private String warna;

    //constructor
    public Aksesoris()
    {
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }
    public Aksesoris(String jenis, String bahan, String warna)
    {
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    //setter
    public void setjenis(String jenis)
    {
        this.jenis = jenis;
    }
    public void setbahan(String bahan)
    {
        this.bahan = bahan;
    }
    public void setwarna(String warna)
    {
        this.warna = warna;
    }
    
    //getter
    public String getjenis()
    {
        return this.jenis;
    }
    public String getbahan()
    {
        return this.bahan;
    }
    public String getwarna()
    {
        return this.warna;
    }


}